lumispy/signals/common_luminescence.py

```diff
@@ -38,42 +39,25 @@
 class CommonLumi:
     """**General luminescence signal class (dimensionless)**"""
 
-    def crop_edges(self, crop_px):
-        """Crop the amount of pixels from the four edges of the scanning
-        region, from out the edges inwards.
-
+    def crop_edges(self, crop_range=None, crop_px=None, rebin_nav=False, **kwargs):
+        """Wrapper around the new crop_edges function.
         Parameters
         ----------
-        crop_px : int
-            Amount of pixels to be cropped on each side individually.
-
-        Returns
-        -------
-        signal_cropped : CommonLuminescence
-            A smaller cropped CL signal object. If inplace is True, the original
-            object is modified and no LumiSpectrum is returned.
+        crop_range : tuple, optional
```

Comment:
Use the same parameter description as for the original function - to allow for easier maintenance, use a docstring template as in the `savetxt` function.